docs/modules/admin/pages/actiontriggers.adoc

```diff
@@ -1,255 +1,184 @@
-  [% FOR circ IN target %]
-    Title: [% circ.target_copy.call_number.record.simple_record.title %]
-    Barcode: [% circ.target_copy.barcode %]
-    Due: [% date.format(helpers.format_date(circ.due_date), '%Y-%m-%d') %]
-    Item Cost: [% helpers.get_copy_price(circ.target_copy) %]
-    Total Owed For Transaction: [% circ.billable_transaction.summary.total_owed %]
-    Library: [% circ.circ_lib.name %]
-  [% END %]
-
-  [% FOR circ IN target %]
-    Title: [% circ.target_copy.call_number.record.simple_record.title %]
-    Barcode: [% circ.target_copy.barcode %]
-    Due: [% date.format(helpers.format_date(circ.due_date), '%Y-%m-%d') %]
-    Item Cost: [% helpers.get_copy_price(circ.target_copy) %]
-    Total Owed For Transaction: [% circ.billable_transaction.summary.total_owed %]
-    Library: [% circ.circ_lib.name %]
-  [% END %]
-
-. Once you are satisfied with your new event trigger, click the _Save_ button located at the bottom of the form.
-
-
-[TIP]
-=========
-A quick and easy way to create new action triggers is to clone an existing action trigger.
-=========
-
-[[creating_alternate_message_templates]]
-=== Creating Alternate Message Templates ===
-indexterm:[Localization, Notices, Localized Notices]
-
-As of version 3.9 there is the ability to create alternate templates for Action Triggers that will generate locale specific out for Action Triggers.  If you send notices in multiple languages, we recommend putting some words to that effect in your notice templates.  The template, message and message title can all be localized.  To use the feature the following UI elements have been added:
+Action Triggers allow administrators to set up actions for specific events,
+useful for notifications such as hold notifications.
 
-* When you double-click on an Event Definition under Notifications / Action Triggers to edit it there will be a tab option for _Edit Alternate Template_ if the reactor is *ProcessTemplate*, *SendEmail*, or *SendSMS*. Note that this feature does not automatically translate existing templates, and an Evergreen administrator must create new alternate templates for each desired locale.
-* In the Patron Registration and Patron Editor screens staff members may select a locale for a patron and edit it in the _Patron Preferred Language_ field.
-* Patrons may set their own locale in the My Account interface off the OPAC by going to _Preferences --> Personal Information_ and setting the _Preferred Language_ field.
+=== Accessing the Notifications / Action Triggers Module ===
 
-The templates used on the Edit Definition tab are the defaults that are used if there are no alternate templates available that match the preferred language.  If alternate templates are available the system will use a locale that is an exact match and then if failing that use one where the language code matches and then fall back to the default one.
+Access the Booking Administration from the top menu bar in the staff client
```

Comment:
```suggestion
Access the Action Triggers Administration from the top menu bar in the staff client
```

Reply:
Lol. I would do that. I'll take care of it. Thanks for reviewing my work carefully as always!

Reply:
@sandbergja Should be all set now.